protected/views/patient/view.php

```diff
@@ -92,6 +92,14 @@
 				</section>
 			<?php endif ?>
 			<?php $this->renderPartial('_patient_details')?>
+      <?php
+        $api = Yii::app()->moduleAPI->get('OETrial');
```

Comment:
Should there be a null check here? That would prevent the code from breaking if OETrial wasn't available.